code/game/objects/items/weapons/extinguisher.dm

```diff
@@ -19,6 +19,8 @@
 	resistance_flags = FIRE_PROOF
 	new_attack_chain = TRUE
 	var/max_water = 50
+	/// We use max_water and max_firefighting_foam to disinguish which extinguishers get Water and which get Firefighting Foam
+	var/max_firefighting_foam = 0
```

Comment:
could you do this by creating a separate subtype instead, and generalizing the values here? so something like:
```dm
/obj/item/extinguisher
	// ...
	var/reagent_id = "water"
	var/reagent_volume = 50

/obj/item/extinguisher/Initialize(mapload)
	// ...
		create_reagents(reagent_volume)
		reagents.add_reagent(reagent_id)

/obj/item/extinguisher/atmospherics
	reagent_id = "firefighting_foam"
	reagent_volume = 65
```